e2e/bundlers/eslint.config.mjs

```diff
@@ -0,0 +1,157 @@
+import babelParser from '@babel/eslint-parser';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import js from '@eslint/js';
+// eslint-disable-next-line spellcheck/spell-checker
+import { FlatCompat as FlatCompatibility } from '@eslint/eslintrc';
+import stylistic from '@stylistic/eslint-plugin';
+import importPlugin from 'eslint-plugin-import';
+import i18N from 'eslint-plugin-i18n';
+import noOnlyTests from 'eslint-plugin-no-only-tests';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compatibility = new FlatCompatibility({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all
+});
+
+export default [
+  {
+    ignores: [
+      'node_modules/**',
+    ],
+  },
+  ...compatibility.extends('devextreme/spell-check'),
+  {
+    plugins: {
+      'no-only-tests': noOnlyTests,
+      i18n: i18N,
+    },
+    settings: {
+      'import/resolver': {
+        node: {
+          extensions: ['.js'],
+        },
+      },
+    },
+  },
+  {
+    ...js.configs.recommended,
+    ...importPlugin.flatConfigs.recommended,
```

Comment:
Please check [this comment](https://github.com/DevExpress/DevExtreme/pull/30845/files#r2320917781)

Reply:
modified it